lblod/frontend-gelinkt-notuleren
# Include snippet button fix

### Overview
See [plugins PR](https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/479) for details.

##### connected issues and PRs:
Plugins PR: https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/479
Jira ticket: [binnenland.atlassian.net/browse/GN-5019](https://binnenland.atlassian.net/browse/GN-5019)

### Setup
N/A

### How to test/reproduce
See [plugins PR](https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/479) for details.

### Challenges/uncertainties
N/A

### Checks PR readiness
